src/schema/v2/homeView/sections/index.ts

```diff
@@ -56,19 +58,21 @@ const sections: HomeViewSection[] = [
   Auctions,
   CuratorsPicksEmerging,
   DiscoverMarketingCollections,
+  DiscoverSomethingNew,
+  ExploreByCategory,
+  ExploreByMarketingCollectionCategories,
```

Comment:
**question:** I haven't seen this order of elements in figma, is this in recent designs?

Reply:
[Explore by](https://www.figma.com/board/kgm28u8rZLh3EfznOuOYSe/Homefeed-Versions?node-id=1368-2111&t=4MLOZEmkTZCkGOfN-4) should be after Hero units and Discover should be after [Recently Viewed](https://www.figma.com/board/kgm28u8rZLh3EfznOuOYSe/Homefeed-Versions?node-id=1368-2085&t=4MLOZEmkTZCkGOfN-4) isn't it?